src/ui/target-select-ui-handler.ts

```diff
@@ -165,6 +177,13 @@ export default class TargetSelectUiHandler extends UiHandler {
     }
   }
 
+  private highlightItems(targetId: number, val: number) : void {
+    const targetItems = this.enemyModifiers.getAll("name", targetId.toString());
+    for (const item of targetItems) {
+      (item as Phaser.GameObjects.Container).setAlpha(val);
```

Comment:
```suggestion
    for (const item of targetItems as Phaser.GameObjects.Container[]) {
      item.setAlpha(val);
```
NIT: Double check if this works since this would be a bit more cleanly/performant than having to cast every iteration